Little util (#56)
virtual-kubelet/systemk: cmd/systemk-volume/README.md

# systemk-volume

This is a helper binary that chowns specific paths *before* the main units start taking over. This
helps to move as much state from systemk into systemd.

> Every pod gets two extra units, one that runs before and one that runs after: they will be called:

Reviewer: ```suggestion
Every pod gets two extra units, one that runs before and one that runs after the execution, named, respectively:
```

> Every pod gets two extra units, one that runs before and one that runs after: they will be called:

Reviewer: It seems implicit that this is enforced, not opt-in. It would help to be more explicit about the intent.
